### PR TITLE
[MM-48637] remove the redundant grid spacing for smaller screen

### PR DESCRIPTION
**Pull request by mhd-sln**
#### Summary
fix the misplaced trial banner and shorter text for trial banner for smaller screens.
edit: the misplaced trial banner was meanwhile fixed in the following [PR](https://github.com/mattermost/mattermost-webapp/pull/11920) and has been pushed to the master.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-48637

#### Screenshots

|  before  |  after  |
|----|----|
| image | Screenshot 2023-01-22 at 23 11 09 |

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by pvev — APPROVED** [review]
LGTM! thanks @mhd-sln :shipit:

File: components/announcement_bar/configuration_bar/configuration_bar.tsx [hunk 1]
```diff
@@ -247,18 +250,22 @@ const ConfigurationAnnouncementBar = (props: Props) => {
...
                         <img
                             className='advisor-icon'
                             src={warningIcon}
                         />
-                        <FormattedMessage
-                            id='announcement_bar.error.trial_license_expiring_last_day'
-                            tagName='strong'
-                            defaultMessage={'This is the last day of your free trial. Purchase a license now to continue using Mattermost Professional and Enterprise features.'}
-                        />
+                        <strong>{viewportBasedMessage}</strong>
```
Comment (julmondragon) [inline components/announcement_bar/configuration_bar/configuration_bar.tsx:268]: nit: Solving the issue by reducing the message text could not work in other languages. Just make sure UX is ok with it.

**Review by julmondragon — COMMENTED** [review]
Thanks @mhd-sln ! Some comments and nits:

File: components/announcement_bar/configuration_bar/configuration_bar.tsx (hunk 1 above)
Comment (mhd-sln) [reply components/announcement_bar/configuration_bar/configuration_bar.tsx:268]: Its Ok. confirmed from @annelieseklein

**Review by julmondragon — APPROVED** [review]

**Comment by annelieseklein** [comment]
@mhd-sln can you please retrigger the banner for me?

**Comment by mhd-sln** [comment]
@annelieseklein the changes are applicable to a self hosted instance. Its abit of work to modify the self hosted spinwick to move the trial expiry day to less than one day. Since, the code changes are small. I will QA defer this.

**Comment by mhd-sln** [comment]
hey @pvev @julmondragon, can you please do one more round of code review ? I have added the test cases [(commit)](https://github.com/mattermost/mattermost-webapp/pull/11996/commits/5268e3e43ab751620b0bd8d8fb2445f78b25dfa0). Thanks!

**Comment by pvev** [comment]
> hey @pvev @julmondragon, can you please do one more round of code review ? I have added the test cases [(commit)](https://github.com/mattermost/mattermost-webapp/pull/11996/commits/5268e3e43ab751620b0bd8d8fb2445f78b25dfa0). Thanks!

sure

**Review by pvev — APPROVED** [review]
LGTM! thanks @mhd-sln :shipit:

**Pull request merged**